packages/react-widgets/src/DropdownList.js

```diff
@@ -496,6 +496,10 @@ class DropdownList extends React.Component {
           key = this.props.open ? 'focusedItem' : 'selectedItem',
           item = list.next(this.state[key], word)
 
+        if(item === this.state[key]) {
+          item = list.prev(this.state[key], word)
```

Comment:
So this will move the last item, then to second to last item, then to the last item, etc. etc.

what you want to do here is if the next item is the current item, start over at the beginning, e.g. `list.next(null, word)`

Reply:
No, actually.

First, it will make the search from *the current position* to *the end* of the list(as it was before).
If there is no result it will male search from *the current position* to *the beginning* of the list

It looks a little bit more efficient for me as we will not make a duplicate search on the second half of the list, but I can update PR with your suggestion if you feel it more suitable.

Reply:
Did you try it? It loops on the last two matching items. I think i'd prefer that the search loops, that's the old behavior and should be what is restored, thanks!

Reply:
yeah, you're right, will update

Reply:
done, not sure about testing - as you don't nest `describe` I've added a similar test with the back direction